db/migrate/20250710102413_populate_payments_with_customer_id.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class PopulatePaymentsWithCustomerId < ActiveRecord::Migration[8.0]
+  def change
+    # rubocop:disable Rails/SkipsModelValidations
+    Payment.where(payable_type: "Invoice")
+      .find_in_batches(batch_size: 1000) do |batch|
+      Payment.where(id: batch.pluck(:id))
+        .update_all("customer_id = (SELECT customer_id FROM invoices WHERE invoices.id = payments.payable_id)")
+    end
+
+    Payment.where(payable_type: "PaymentRequest")
+      .find_in_batches(batch_size: 1000) do |batch|
+      Payment.where(id: batch.pluck(:id))
+        .update_all("customer_id = (SELECT customer_id FROM payment_requests WHERE payment_requests.id = payments.payable_id)")
+    end
+    # rubocop:enable Rails/SkipsModelValidations
+  end
+end
```

Comment:
I think this should be a job. What's one more job in this folder? 😄 
https://github.com/getlago/lago-api/tree/main/app/jobs/database_migrations

Reply:
Added it there, and also added a "universal" task inpired by @vincent-pochet:
https://github.com/getlago/lago-api/pull/3961/files#diff-41329f763f577b8b1af65e00c1b6756e05281527f0bcc73a80cb306988e54deb
(I had a discussion with one of our customers that if the command to execute required jobs was always the same, they might have added it to their deployment script)

Reply:
awesome!